tools/src/versioning/android/index.ts

```diff
@@ -556,48 +585,75 @@ async function prepareReanimatedAsync(version: string): Promise<void> {
   await removeLeftoversFromGradle();
 }
 
+async function exportReactNdks() {
+  const versionedRN = path.join(versionedReactAndroidPath, '..');
+  await spawnAsync(`./gradlew :ReactAndroid:packageReactNdkLibs`, [], {
+    shell: true,
+    cwd: versionedRN,
+    stdio: 'inherit',
+  });
+}
+
+async function exportReactNdksIfNeeded() {
+  const ndksPath = path.join(versionedReactAndroidPath, 'build', 'react-ndk', 'exported');
+  const exists = await fs.pathExists(ndksPath);
+  if (!exists) {
+    await exportReactNdks();
+    return;
+  }
+
+  const exportedSO = await glob(path.join(ndksPath, '**/*.so'));
+  if (!exportedSO) {
+    await exportReactNdks();
+  }
```

Comment:
Hmm, can `glob` resolve to something different than an array (which is always truthy)? I would say it returns empty array if nothing has been found.